src/bench/ss_bench/operations/write_batch.rs

```diff
@@ -98,13 +98,6 @@ impl Operations {
             if let Some(context) = context {
                 if let Some(task) = self.meta_client.get_compact_task().await {
                     Compactor::compact(context.clone(), task).await;
-                    let last_pinned = context.local_version_manager.get_version().unwrap();
-                    let version = self
-                        .meta_client
-                        .pin_version(last_pinned.id())
-                        .await
-                        .unwrap();
-                    context.local_version_manager.try_set_version(version);
```

Comment:
Why pinning is removed here?

Reply:
Local versions is being updated periodically by another task in `HummockStorage`, so this code snippet is not required.

Reply:
This code snippet originally attempts to ensure the version after compaction is available locally during benchmark.
It doesn't use pin_version properly but can be a temporary workaround for now.
I've resumed it and added some comments.